### PR TITLE
41 update django version

### LINKED ISSUE #41: Update for Opal 0.10.0 compatibility
### updating opal-lab
- [x] create a new version of opal-labs 
- [x] change the default version of opal-labs

#### On a new branch
- [x] change the requirements.txt
- [x] update the urls and requirements into the django 10 format [e.g.]https://github.com/openhealthcare/opal-observations/pull/40/files)
- [x] changelog
- [x] make sure tests run on opal 10
- [x] create a pull request

### PR DESCRIPTION
**Pull request by pacharanero**
closes #41 

updated for django 1.10.8, django-reversion 1.10.2 and opal 0.10.0

### CODE REVIEW

File: lab/admin.py [hunk 1]
```diff
@@ -1,10 +1,11 @@
-import reversion
+from reversion.admin import VersionAdmin
+from reversion import revisions as reversion
```
Comment (fredkingham) [inline lab/admin.py:2]: @pacharanero do we need this?

File: lab/templatetags/lab_tags.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 from django import template
-from django.template import loader, Context
+import django.template.loader as loader
```
Comment (fredkingham) [inline lab/templatetags/lab_tags.py:2]: The below is more traditional, that does work but I've never seen it before in the wild!

from django.template import loader

File: lab/admin.py (hunk 1 above)
Comment (pacharanero) [reply lab/admin.py:2]: Probably not, in retrospect. I changed that when I was having an problem with VersionAdmin but it turned out to be caused by a different issue. I will revert, test and push.

Comment (pacharanero) [reply lab/admin.py:2]: after Slack discussion we decided to leave it as it is

File: lab/templatetags/lab_tags.py (hunk 2 above)
Comment (pacharanero) [reply lab/templatetags/lab_tags.py:2]: I've now fixed this one

**Pull request merged**